mtgjson4/compile_mtg.py

```diff
@@ -84,36 +84,54 @@ def build_output_file(sf_cards: List[Dict[str, Any]], set_code: str) -> Dict[str
 def uniquify_duplicates_in_set(cards: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
     """
     For cards with multiple printings in a set, we need to identify
-    them against each other. We will add (a), (b), ... to the end
+    them against each other.
+    For silver border sets, we will add (a), (b), ... to the end
```

Comment:
Why do this for silver sets?

Reply:
Because Everythingamagigs, etc from Unstable are all _different_ cards compared to different printings

Reply:
The one consistent thing in magic is that nothing is consistent

Reply:
In black border magic, two cards cannot have the same name if they don't have the same rules text. This was thrown out for Unstable, so I added a blanket clause for all un-sets